HeterogeneousCore/CUDAServices/interface/supportedCudaDevices.h

```diff
@@ -0,0 +1,8 @@
+#ifndef HeterogeneousCore_CUDAServices_interface_supportedCudaDevices_h
+#define HeterogeneousCore_CUDAServices_interface_supportedCudaDevices_h
+
+#include <map>
+
+std::map<int, std::pair<int, int>> supportedCudaDevices(bool reset = true);
```

Comment:
```suggestion
std::map<int, std::pair<int, int>> supportedCUDADevices(bool reset = true);
```
would be more consistent with spelling elsewhere.

Reply:
yes, I will make the change